### PR TITLE
[ANDWL-324] kado money add

### PR DESCRIPTION
**Pull request by kimheeje12**

### CODE REVIEW

File: app/src/main/java/wannabit/io/cosmostaion/base/BaseActivity.java [hunk 1]
```diff
@@ -931,19 +934,42 @@ else if (result.taskType == TASK_FETCH_MINTSCAN_CW20_ASSETS) {
...
-                "USD", view -> onStartMoonpaySignature("usd"), null,
-                "EUR", view -> onStartMoonpaySignature("eur"), null,
-                "GBP", view -> onStartMoonpaySignature("gbp"), null);
+                "USD", view -> onStartMoonPaySignature("usd"), null,
+                "EUR", view -> onStartMoonPaySignature("eur"), null,
+                "GBP", view -> onStartMoonPaySignature("gbp"), null);
+    }
+
+    public void onShowBuyKado() {
+        String query = "?apiKey=" + getString(R.string.moon_pay_public_key);
```
Comment (soaryong-c) [inline app/src/main/java/wannabit/io/cosmostaion/base/BaseActivity.java:955]: moon_pay_public_key

File: app/src/main/java/wannabit/io/cosmostaion/base/BaseConstant.java [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import java.math.BigDecimal;
 
 public class BaseConstant {
-    public final static boolean SUPPORT_MOONPAY         = true;
+    public final static boolean SUPPORT_PAY             = true;
```
Comment (soaryong-c) [inline app/src/main/java/wannabit/io/cosmostaion/base/BaseConstant.java:6]: not need.

File: app/src/main/java/wannabit/io/cosmostaion/widget/mainWallet/WalletPriceHolder.java [hunk 3]
```diff
@@ -54,32 +57,44 @@ public void onBindHolder(@NotNull MainActivity mainActivity) {
...
+                itemBuyCoinTv.setText(R.string.str_buy_kujira);
+            }
         } else {
             itemBuyLayer.setVisibility(View.GONE);
         }
 
         itemBuyCoinBtn.setOnClickListener(v -> {
             if (mainActivity.mAccount.hasPrivateKey) {
-                mainActivity.onShowBuySelectFiat();
+                if (mainActivity.mBaseChain.equals(COSMOS_MAIN) || mainActivity.mBaseChain.equals(BNB_MAIN) || mainActivity.mBaseChain.equals(KAVA_MAIN)) {
```
Comment (soaryong-c) [inline app/src/main/java/wannabit/io/cosmostaion/widget/mainWallet/WalletPriceHolder.java:81]: need refactoring

**Review by soaryong-c — CHANGES_REQUESTED** [review]

File: app/src/main/java/wannabit/io/cosmostaion/base/BaseActivity.java [hunk 4]
```diff
@@ -936,14 +939,43 @@ public void onShowBuyWarnNoKey() {
...
+            query = query + "&network=osmosis";
+        } else if (mBaseChain.equals(JUNO_MAIN)) {
+            query = query + "&network=juno";
+        } else if (mBaseChain.equals(KUJIRA_MAIN)) {
+            query = query + "&network=kujira";
+        }
+        final String SendQuery = query + "&onToAddress=" + mAccount.address;
+        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.url_kado_money) + SendQuery));
+        startActivity(intent);
+        Toast.makeText(getBaseContext(), R.string.error_network_error, Toast.LENGTH_SHORT).show();
```
Comment (Kwonhyukjoon) [inline app/src/main/java/wannabit/io/cosmostaion/base/BaseActivity.java:975]: need this?

Comment (kimheeje12) [reply app/src/main/java/wannabit/io/cosmostaion/base/BaseActivity.java]: Thank you

**Review by soaryong-c — APPROVED** [review]
👍

**Comment by soaryong-c** [comment]
resolve conflicts please

**Review by soaryong-c — APPROVED** [review]
👍

**Review by soaryong-c — APPROVED** [review]
👍

**Pull request merged**